src/components/MonthlyGood/MonthlyGoodSelectorWrapper.vue

```diff
@@ -80,19 +82,26 @@ export default {
 			// set offset
 			this.mgStickBarOffset = offsetHeight;
 		},
+		determineIfMobile() {
+			this.isMobile = document.documentElement.clientWidth < 480;
```

Comment:
This  feels kind of hacky, but since both MG selectors, desktop and mobile need to listen to the root event from contentful, I need vue to actually remove it from the DOM with v-if instead of just hiding it with CSS utility classes. 

Possible opportunity for a mixin or utility function?